homeserver.yaml

```diff
@@ -0,0 +1,37 @@
+# Configuration file for Synapse.
+#
+# This is a YAML file: see [1] for a quick introduction. Note in particular
+# that *indentation is important*: all the elements of a list or dictionary
+# should have the same indentation.
+#
+# [1] https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html
+#
+# For more information on how to configure Synapse, including a complete accounting of
+# each option, go to docs/usage/configuration/config_documentation.md or
+# https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html
+server_name: "my.matrix.host"
+pid_file: /data/homeserver.pid
+listeners:
+  - port: 8008
+    tls: false
```

Comment:
Who needs encryption? 😉

Reply:
I know, right? :) The path taken here is to have synapse listen locally and terminate TLS on the ingress. So that port shouldn't be exposed and clients shouldn't be able to talk to synapse directly.